components/server/ee/src/user/stripe-service.ts

```diff
@@ -85,7 +87,9 @@ export class StripeService {
             email: User.getPrimaryEmail(user),
             name: userName ? `${userName} (${team.name})` : team.name,
             metadata: {
+                // teamId is deprecated, use attributionId where possible
                 teamId: team.id,
```

Comment:
Nit: Same as above -- the only place where the `teamId` metadata is used is in `findCustomerByTeamId` above. If you update this method, you can remove the now useless `teamId` metadata.

```suggestion
```

Reply:
> We cannot yet update the find queries to use this field as we have customers in Stripe without the attributionID property.

We cannot remove it yet, lookup queries would fail on existing customers.

Reply:
The reason it stays on the create is consistency, we want to phase out teamId, but that doesn't mean we can stop setting it, due to existing queries.